build/buildPythonSdkByVersion.sh

```diff
@@ -36,14 +36,16 @@ buildPythonfromSource()
         apt-get update && \
         apt-get upgrade -y && \
         DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+            build-essential \
```

Comment:
Could you please clarify why we're adding build-essential and libgeos-dev? I'm just curious to understand. :)

Reply:
Hi Dan, I removed these dependencies from the template.Dockerfile because it'll apply to all python versions if we put in the template.Dockerfile(It's also the reason why it causes other image size increase). But they're still required dependencies for python 3.10 so I put them here only applies to 3.10.